examples/basics/demo_custom_prior.py

```diff
@@ -17,7 +17,7 @@
 from deepinv.optim.optimizers import optim_builder
 from deepinv.training import test
 from torchvision import transforms
-from deepinv.utils.demo import load_dataset
+from deepinv.utils.demo import load_dataset, get_data_home
```

Comment:
why do we need to import get_data_home?

Reply:
Good catch, I changed so many I got lost in the process I guess :)
```suggestion
from deepinv.utils.demo import load_dataset
```